homeassistant/components/sensor/ted5000.py

```diff
@@ -22,13 +22,14 @@
 
 DEFAULT_NAME = 'ted'
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=3)
```

Comment:
This should be set to a reasonable default. Polling a device every three seconds seems a bit overkill for sensor values.

It should be the other way around, if the user want fast update then he/she should overwrite it.

Reply:
If this value is set to 10s, no matter the override you put, it won't call it faster than 10s